src/Parser/Warrior/Arms/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,19 @@
+import SPELLS from 'common/SPELLS';
+
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+// eslint-disable no-unused-vars
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
+    ...CoreCastEfficiency.CPM_ABILITIES,
+    {
+      spell: SPELLS.BATTLE_CRY,
+      category: CastEfficiency.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 60,
+      noSuggestion: true,
+    },
```

Comment:
Potentially want to work to expand this area within the first PR just to give some additional context for the user on what they are casting, or casts they are missing.